index.js

```diff
@@ -70,6 +70,11 @@ module.exports = async (opts) => {
     style = { },
     inject = { },
     puppeteerOptions = { },
+    ffmpegOptions = {
+      crf: 20,
+      profileVideo: "main",
```

Comment:
Make sure you always run `yarn test` before submitting your PR. In this case, there are some eslint errors related to using double quotes instead of single quotes. This isn't something that affects the program at all, but using a consistent style and conforming to the original author's config is important for long term maintainability.

Reply:
Ok, no problem, I fixed the errors.
It might be nice to add one line for the test in the readme to be sure :)